src/Datadog.Trace/Logging/LibLogScopeEventSubscriber.cs

```diff
@@ -147,6 +170,15 @@ private void SetCorrelationIdentifierContext(ulong traceId, ulong spanId)
             try
             {
                 // TODO: Debug logs
+                _contextDisposalStack.Push(
+                    LogProvider.OpenMappedContext(
```

Comment:
@brettlangdon since this codebase will be foreign to you, we subscribe to events that fire when new spans are made active, and we use this `LogProvider` abstraction to feed all of the span properties into the logging context at once. We subsequently remove them all at once